libindex/migrations/04-foreign-key-cascades.sql

```diff
@@ -0,0 +1,165 @@
+-- Add CASADEs on all foreign-key relations.
```

Comment:
Do we want to lock the tables? I'm trying to think of someone is cycling a bunch of indexer pods with new versions while the old ones are still working.

Reply:
according to the docs, the PostgresQL migrator takes an advisory lock